.eslintrc

```diff
@@ -3,7 +3,10 @@
     "oclif",
     "oclif-typescript"
   ],
-  "rules": {
-    "unicorn/no-abusive-eslint-disable": 0
-  }
+  "overrides": [{
+    "files": [ "./test/**/*" ],
```

Comment:
It does lint them, it just overrides the `"node/no-extraneous-import"` rule for test files because the fixtures have their own `package.json`s that have no dependencies so eslint complains that you shouldn't use dependencies that belong to some other package.

Reply:
Oh I see/read!